src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -53,42 +52,26 @@ public class SnapshotBuilderTestData {
                       .id(1)
                       .name("Year of birth")
                       .kind(SnapshotBuilderProgramDataOption.KindEnum.RANGE)
-                      .min(1900)
-                      .max(2023),
+                      .tableName("person")
+                      .columnName("year_of_birth"),
```

Comment:
So the refactor looks good and makes sense. My question is does this PR accomplish the goal of the original ticket or is that still to come - depending on the new stubbed endpoint and an additional call in the UI?

Reply:
That is still to come, we will need to make the call from the UI to get the information.

Reply:
Okay makes sense. Do you think we should update the ticket and PR name?

Reply:
☝️ I like this idea.  I find myself using `git blame` a lot to track back to past PRs for context, and appreciate when the titles, descriptions, and comment threads make any evolution in approach or problem solved clear.

More specifically, I'm currently having a hard time figuring out what changes were necessary for the original goal of supporting the range slider, and what changes were refactors / opportunities for improvement followed through on while you were working in this space.